tests/stress_tests/scripts/get_testdata.py

```diff
@@ -162,8 +161,52 @@ def main():
         log.warning("List of models defined in config but not available in OMZ: {}"
                     .format(",".join(set(models_from_cfg) - models_to_run)))
 
+    # update test config with Open Model Zoo info
+    info_dumper_path = omz_path / "tools" / "downloader" / "info_dumper.py"
+    cmd = "{executable} {info_dumper_path} --list {models_list_path}"\
+        .format(executable=sys.executable, info_dumper_path=info_dumper_path,
+                models_list_path=models_list_path)
+    out = subprocess.check_output(cmd, shell=True, universal_newlines=True)
+    models_info = json.loads(out)
+
+    fields_to_add = ["framework", "subdirectory"]
+    models_root = test_conf_root.find("models")
+    for model_rec in models_root:
+        if model_rec.attrib.get("source") == "omz" and "name" in model_rec.attrib:
+            if model_rec.attrib["name"] not in models_to_run:
+                # remove models from test config which aren't available in OMZ
+                models_root.remove(model_rec)
+            else:
+                # Open Model Zoo ensures name uniqueness of every model
+                models_info_rec = next(iter([rec for rec in models_info if rec["name"] == model_rec.attrib["name"]]))
+                info_to_add = {key: models_info_rec[key] for key in fields_to_add}
+                model_rec.attrib.update(info_to_add)
+                model_rec.attrib["precision"] = PRECISION
+                model_rec.attrib["path"] = str(
+                    Path(model_rec.attrib["subdirectory"]) / PRECISION / (model_rec.attrib["name"] + ".xml"))
+                model_rec.attrib["full_path"] = str(
+                    args.omz_irs_out_dir / model_rec.attrib["subdirectory"] / PRECISION / (model_rec.attrib["name"] + ".xml"))
+    test_conf_obj.write(args.test_conf)
+
+    # prepare models
+    cmd = '{downloader_path} --list {models_list_path}' \
+          ' --num_attempts {num_attempts}' \
+          ' --output_dir {models_dir}' \
+          ' --cache_dir {cache_dir}' \
+          ' --jobs {jobs_num}'.format(downloader_path=downloader_path, models_list_path=models_list_path,
+                                      num_attempts=OMZ_NUM_ATTEMPTS,
+                                      models_dir=args.omz_models_out_dir,
+                                      cache_dir=args.omz_cache_dir,
+                                      jobs_num=DOWNLOADER_JOBS_NUM)
+
     run_in_subprocess(cmd, check_call=not args.skip_omz_errors)
 
+    # Open Model Zoo doesn't copy downloaded IRs to converter.py output folder where IRs should be stored.
+    # Do it manually to have only one folder with IRs
+    for ir_src_path in args.omz_models_out_dir.rglob("*.xml"):
```

Comment:
Some errors here:
1. Copy of ".xml" only
2. Need to copy additional files like `faster_rcnn_inception_v2_coco_2018_01_28/pipeline.config`

Solutions:
1. Avoid using `args.omz_irs_out_dir` and use only one output directory

Reply:
There are no issues because of  `shutil.copytree` (except duplication of original networks)